src/components/calcite-input-message/calcite-input-message.e2e.ts

```diff
@@ -1,5 +1,6 @@
 import { newE2EPage } from "@stencil/core/testing";
 import { accessible, renders } from "../../tests/commonTests";
+import bananaShape from "../calcite-icon/assets/banana16.json";
```

Comment:
🍌  🥳

Reply:
You may have issues importing from here (it depends on the copy icons task already completing) I think you could import this directly from them dep like:

```ts
import { banana16 } from "@esri/calcite-ui-icons/js/banana16.js";
```

Reply:
excellent thank you for the tip @paulcpederson

Reply:
@paulcpederson unfortunately I couldn't get the module import route to work for me :/